### DISCUSSION TITLE
How can i see an entry with the publish date on today

### DISCUSSION THREAD

**Discussion by Oleafeon** [statamic/cms#4304] [Q&A]
## Bug Description
When i set the publish date for my collection with the date_behavior of: 
```
date_behavior:
  past: private
  future: public
```
and an entry has it on today it wont show up on the website, how can i make it so that it does do that.
So what i want to see on the frontend is every entry with the date of today or in the future

## Environment
**Statamic version:** 3.2.5

**PHP version:** 7.4

**Install method (choose one):**
- Fresh install from `statamic/statamic`

**Comment by Oleafeon** [comment]
Possible fix for now is to add the time to the field and put the time on 23:59

**Comment by wm-simon** [comment]
I had this issue too. Perhaps this helps you:
https://github.com/statamic/cms/issues/3184 

I solved it on this way:
`date:is_after="{today | modify_date:-1 day}"`

**Comment by eliasu** [comment]
Why not forget about the **date_behavior** and fix that within the antlers? Something like:
```
{{collection:dates}}
    {{ if date-field | is_future || date-field | is_today }}
{{/collection:dates}}
```
